extutils/imgproc/apng2gif.py

```diff
@@ -1,22 +1,93 @@
-from dataclasses import dataclass
+"""
+Module to convert ``apng`` to ``gif``.
+"""
+import io
+from dataclasses import dataclass, field
 from fractions import Fraction
-import glob
 import os
 import shutil
-import subprocess
 from tempfile import TemporaryDirectory
 import time
 from typing import Any, Tuple, List, Optional
 from zipfile import ZipFile
 
 from PIL import Image
 
-__all__ = ["convert", "ConvertResult"]
+from .apng2png import extract_frames
+
+__all__ = ["convert", "ConvertResult", "ConvertOpResult"]
 
 _IDX_CLR_BACKGROUND = 255
 _IDX_CLR_TRANSPARENT = 255
 
 
+class ConvertOpResult:
+    """
+    Result of an operation during the conversion.
+    """
+
+    def __init__(self):
```

Comment:
Codacy found an issue: [Missing docstring in __init__ (D107)](https://app.codacy.com/manual/RaenonX/Jelly-Bot/pullRequest?prid=6015490)